pkg/monitoring/metrics.go

```diff
@@ -40,6 +47,42 @@ func SetReadyGauge(isReady bool) {
 	}
 }
 
+func setKubemacpoolDeployedGauge(isDeployed bool) {
+	if isDeployed {
+		kmpDeployedGauge.Set(1)
+	} else {
+		kmpDeployedGauge.Set(0)
+	}
+}
+
+func ResetMonitoringComponents() error {
+	SetReadyGauge(false)
+	setKubemacpoolDeployedGauge(false)
+	return nil
```

Comment:
same deal here, i think return nil for a function that doesn't do anything else is totally unneeded,
its ok that a function won't return a value

maybe worth that the SetReadyGauge will be on other PR / commit, but i won't insist

Reply:
in this case I tend to agree there is no need for error in the function. will remove